card/src/main/java/com/adyen/checkout/card/NewCardDelegate.kt

```diff
@@ -79,10 +79,18 @@ class NewCardDelegate(
         }
     }
 
+    override fun validateSocialSecurityNumber(socialSecurityNumber: String): FieldState<String> {
+        return CardValidationUtils.validateSocialSecurityNumber(socialSecurityNumber)
+    }
+
     override fun isCvcHidden(): Boolean {
         return cardConfiguration.isHideCvc
     }
 
+    override fun getSocialSecurityNumberVisibility(): SocialSecurityNumberVisibility? {
```

Comment:
Marc said this is not required in stored cards for now, and while testing I saw a weird behavior where the cursor would jump back to the cvc field.